qlf/dashboard/bokeh/exposures/main.py

```diff
@@ -11,18 +11,22 @@
 
 # Get the list of exposures
 exposures = get_exposures()
-print(exposures)
 
-# By default display the last one on the interface
-expid = exposures['expid'][-1]
-flavor = exposures['flavor'][-1]
+expid = None
 
-# Page title reflects the selected exposure
-title = Div(text="<h3>Exposure ID {} ({})</h3>".format(expid, flavor))
+if exposures['expid']:
+    exposures['expid'] = sorted(exposures['expid'])
 
-# Here we configure a 'slider' to change the exposure
-slider = Slider(start=exposures['expid'][0], end=expid,
-                value=expid, step=1, title="EXPID")
+    # By default display the last one on the interface
+    expid = exposures['expid'][-1]
```

Comment:
I would define two variables here `start = exposures['expid'][0]` and `end = exposures['expid'][-1]` and use later.